utilities/utils/response_windows.go

```diff
@@ -49,3 +51,38 @@ func getIP(inspect types.ContainerJSON, cache *cache.Cache) (string, error) {
 	hijack.Close()
 	return ip, nil
 }
+
+func setupDNS(containerID string) error {
+	command := []string{"powershell", "Get-NetAdapter", "|", "Set-DnsClientServerAddress", "-ServerAddresses"}
+	addressesArg := ""
+	for i, ip := range strings.Split(config.DNSAddresses(), ",") {
+		if i > 0 {
+			addressesArg += ","
+		}
+		addressesArg += fmt.Sprintf("'%s'", ip)
+	}
+	command = append(command, fmt.Sprintf("(%s)", addressesArg))
+	return createAndStart(containerID, command)
+}
+
+func createAndStart(containerID string, command []string) error {
```

Comment:
This function name doesn't accurately represent what it does to me. Could you change it to something like doExec (or eliminate it entirely and just put the logic directly into the calling method)?